eZ/Publish/Core/REST/Client/Output/ValueObjectVisitor/ViewInput.php

```diff
@@ -29,15 +35,14 @@ public function visit(Visitor $visitor, Generator $generator, $data)
         $generator->endValueElement('identifier');
 
         if ($data->locationQuery !== null ) {
-            $queryElementName = 'LocationQuery';
+            $queryElementName = 'locationQuery';
         } elseif ($data->contentQuery !== null ) {
-            $queryElementName = 'ContentQuery';
+            $queryElementName = 'contentQuery';
```

Comment:
This is REST client side property it seems, so not an issue, but indeed confusing when reviewing :)